tests/e2e/unified_test.go

```diff
@@ -169,6 +174,11 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 				{
 					_, opt := loadFixture(testgcp.GCPProject{ProjectID: "test-skip", ProjectNumber: 123456789})
 					create.MaybeSkip(t, fixture.Name, opt.Create)
+					if testPause && containsCCOrCCC(opt.Create) {
```

Comment:
I think this is fine for now!  Ideally we would modify the CC/ CCC to turn `actuationMode: Paused` for paused tests but I can take that on.

Reply:
Makes sense. My goal is to unblock this PR reasonably.